integration/ct_integration_test.go

```diff
@@ -306,6 +306,44 @@ func TestCTIntegration(t *testing.T) {
 		t.Errorf("\tuploaded:  %s", hex.EncodeToString(tbs))
 		t.Errorf("\tretrieved: %s", hex.EncodeToString(ts.PrecertEntry.TBSCertificate))
 	}
+
+	// Stage 14: get an inclusion proof for the precert.
+	fmt.Printf("Inclusion proof leaf @ %d -> root %d = ", precertSCT.Timestamp, sthN1.TreeSize)
+	// Calculate leaf hash =  SHA256(0x00 | tls-encode(MerkleTreeLeaf))
+	issuer, err := x509.ParseCertificate(prechain[1].Data)
+	if err != nil {
+		t.Fatalf("failed to parse issuer certificate: %v", err)
+	}
+	leaf = ct.MerkleTreeLeaf{
+		Version:  ct.V1,
+		LeafType: ct.TimestampedEntryLeafType,
+		TimestampedEntry: &ct.TimestampedEntry{
+			Timestamp: precertSCT.Timestamp,
+			EntryType: ct.PrecertLogEntryType,
+			PrecertEntry: &ct.PreCert{
+				IssuerKeyHash:  sha256.Sum256(issuer.RawSubjectPublicKeyInfo),
+				TBSCertificate: tbs,
+			},
+			Extensions: precertSCT.Extensions,
+		},
+	}
+	leafData, err := tls.Marshal(leaf)
+	if err != nil {
+		t.Fatalf("tls.Marshal(precertLeaf)=nil,%v", err)
```

Comment:
[Kat suggested](https://github.com/google/certificate-transparency/pull/1325#discussion_r85788610) this style, and I've consistently used it in other places.